scipy/stats/tests/test_fit.py

```diff
@@ -232,11 +254,13 @@ def test_fitstart(distname, shapes):
     assert dist._argcheck(*guess[:-2])
 
 
-def assert_nllf_less_or_close(dist, data, params1, params0, rtol=1e-7, atol=0):
-    nllf1 = dist.nnlf(params1, data)
-    nllf0 = dist.nnlf(params0, data)
-    if not (nllf1 < nllf0):
-        np.testing.assert_allclose(nllf1, nllf0, rtol=rtol, atol=atol)
+def assert_nlff_less_or_close(dist, data, params1, params0, rtol=1e-7, atol=0,
+                              nlff_name='nnlf'):
```

Comment:
Now this accepts the name of a negative log fitness function to assess the quality of the fit. For MLE, we call the negative log likelihood function to assess the fit; for MPE, we call the negative log product spacing function.